qa-tests-backend/src/main/groovy/util/Helpers.groovy

```diff
@@ -128,15 +133,20 @@ class Helpers {
 
     // collectImageScanForDebug(image) - a best effort debug tool to get a complete image scan.
     static void collectImageScanForDebug(String image, String saveName) {
-        if (!Env.IN_CI) {
-            log.info "Won't collect image scans when not in CI"
+        if (!Env.IN_CI && !Env.GATHER_QA_TEST_DEBUG_LOGS) {
+            log.info "Won't collect image scans without CI=true or GATHER_QA_TEST_DEBUG_LOGS=true"
+            return
+        }
+
+        if (Env.QA_TEST_DEBUG_LOGS == "") {
+            log.info "Won't collect image scans when QA_TEST_DEBUG_LOGS is not set"
```

Comment:
This conditions are in two places how about extracting them to a function that will return bool if logs should be collected?

Reply:
Yeah, I'll do that.